### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
 
 echo "[*] Downloading files..."
-cat <<EOF > urls.txt
-https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz
-https://services.gradle.org/distributions/gradle-5.0-bin.zip
-https://ghidra-sre.org/ghidra_9.2.1_PUBLIC_20201215.zip
-https://github.com/Cxbx-Reloaded/XbSymbolDatabase/releases/latest/download/XbSymbolDatabase.zip
-EOF
-cat urls.txt | xargs -n 1 -P 10 wget --no-verbose
+wget -nv https://corretto.aws/downloads/latest/${CORRETTO_ARCHIVE}	\
```
Comment (mborgerson) [inline build.sh:13]: * This wonky looking `xargs` structure I had is here to let these packages download in parallel. Let's prefer to keep it
* Let's be consistent with the URL-in-environment-variable method as you've done for `GHIDRA_URL`

**Review by mborgerson — COMMENTED** [review]
See comments; please fix merge conflicts

File: build.sh (hunk 1 above)
Comment (Margen67) [reply build.sh:13]: Is there any particular reason it's 10 processes?

Comment (mborgerson) [reply build.sh:13]: No particular reason. Just need a number >= number of URLs

Comment (Margen67) [reply build.sh]: I changed it to the size of the array.

**Comment by mborgerson** [comment]
Thanks

**Pull request merged**